packages/utils/src/CopyClipboardUtils.ts

```diff
@@ -0,0 +1,43 @@
+class CopyClipboardUtils {
```

Comment:
I would just rename this file `ClipboardUtils`, no need to specify Copy only.

I know this is how we've done Utils before, but I've kind of changed my ways as I've learned. Instead of using a class and doing static methods, do something like we do in `SessionUtils` where we export each function. So in this case, it would be something like:

```
export function copyToClipboard(...) { ... }

export function copyToClipboardExecCommand(...) { ... }
```
You don't even need an export Default; if another file wants to import all the functions, they can do: `import * as ClipboardUtils from './ClipboardUtils'`. But most of the time it'll just be `import { copyToClipboard } from './ClipboardUtils'` or something.

Some discussions on this topic:
https://www.typescriptlang.org/docs/handbook/modules.html#export-as-close-to-top-level-as-possible
https://stackoverflow.com/questions/29893591/es6-modules-export-single-class-of-static-methods-or-multiple-individual-method/29895235#29895235
https://olegvaraksin.medium.com/what-is-the-best-way-to-write-utilities-in-typescript-e3cae916fe30